syscalls/src/sysvar.rs

```diff
@@ -17,6 +17,14 @@ fn get_sysvar<T: std::fmt::Debug + SysvarSerialize + Clone>(
             .sysvar_base_cost
             .saturating_add(size_of::<T>() as u64),
     )?;
+
+    if var_addr >= ebpf::MM_INPUT_START
```

Comment:
Is it possible to have an address lower than `MM_INPUT_START`? Wouldn't that lead to an access violation?